src/test/java/com/productdock/library/rental/scheduled/CancelReservationShould.java

```diff
@@ -58,14 +58,14 @@ class CancelReservationShould {
     private static final Collection<RentalRecordEntity> ANY_RENTAL_RECORD_COLLECTION = List.of(FIRST_RECORD, SECOND_RECORD);
 
     @BeforeEach
-    public void setUp(){
-        ReflectionTestUtils.setField(cancelReservation, DELAY_FIELD, 345600);
-        ReflectionTestUtils.setField(cancelReservation, DAY_DURATION_FIELD, 86400);
+    public void setUp() {
+        given(reservationHoldPolicy.getTimeUnit()).willReturn(TimeUnit.DAYS);
```

Comment:
When setting up data structures in tests (where no logic is expected), it is ok to make concrete instance of the class. reservationHoldPolicy = new ReservationHoldPolicy()... setTimeUnit(...), setLimit(...)